zephyr/wrapper.c

```diff
@@ -136,6 +151,8 @@ void *rmalloc(enum mem_zone zone, uint32_t flags, uint32_t caps, size_t bytes)
 {
 	if (zone_is_cached(zone))
 		return heap_alloc_aligned_cached(&sof_heap, 0, bytes);
+	else
+		return heap_alloc_aligned(&sof_heap, 8, bytes);
```

Comment:
This line "reliably" crashes my Up Squared board on the second aplay command every time, see partial revert PR #4472

git bisect found this commit and then it was easy to identify this line because it seems to be the only one not IMX-specific.

Cc: @lyakh , @lgirdwood , @andyross

Reply:
I've added a pull request to fix this, for both i.MX and Up Squared: #4477 
The PR #4472 breaks i.MX.